docs/source/concept_models/vivarium_sanofi_multiple_myeloma/concept_model.rst

```diff
@@ -305,13 +305,11 @@ In this model, we implement risk exposures for simulants upon entry to the MM st
 
-Notably, we only have risk factor exposure distribution data among newly diagnosed patients. Due to the differential survival rates among the different risk exposure groups, we do not expect that the risk exposure distribution among relapsed and refractory multiple myeloma patients to be the same as among newly diagnosed patients. Due to our lack of data to inform risk exposure initialization probabilities among RRMM patients at the beginning of the simulation, we will model a "burn-in" period prior to the official time-frame of the simulation (2021-2026) in which all MM patients are initialized to the first MM state, allowing these risk exposure distributions to shift along with disease progression according to the risk effects described in the next section.
+Notably, we only have risk factor exposure distribution data among newly diagnosed patients. Due to the differential survival rates among the different risk exposure groups, we do not expect that the risk exposure distribution among relapsed and refractory multiple myeloma patients to be the same as among newly diagnosed patients. Due to our lack of data to inform risk exposure initialization probabilities among RRMM patients at the beginning of the simulation, we will model a "burn-in" period prior to the official time-frame of the simulation (2021-2026) in which all MM patients are initialized to the first MM state, allowing these risk exposure distributions to shift along with disease progression according to the risk effects described in the next section. Details are described in the :ref:`multiple myeloma cause model document<2019_cancer_model_multiple_myeloma>`. 
 
-.. todo::
-
-  Add more details on the burn-in period... to be expanded upon in the retreatment section
+After the initial 10 burn-in period run, prevalent risk exposure distributions for race, cytogenetic risk, and renal function should be recorded for each MM cause model state. In order to achieve this, person-time stratified by risk exposure and multiple myeloma cause model state is required as a simulation output. Given our assumpiton of independence of risk exposures, the prevalent risk exposure distributions should be evaluated at the population level; however, the age risk exposure distribution should be stratified by GBD 5 year-age group. For future runs in which a burn-in period is not run and simulants are initialized into RRMM states rather than solely the first MM state, simulants should be assigned with risk exposures for race/cytogenetic risk/renal function at diagnosis with probabilities corresponding to the recorded risk exposure distribution specific to the MM cause model state that the simulant is initialized into. 
```

Comment:
> After the initial 10 burn-in period run

10 year? And if 10 years, is that a new definition? My understanding is that burn-in starts before 1/1/2016.

Reply:
So there's going to be two separate burn ins... the first is to get the correct initialization for the cause model and risk exposures, starting by initializing all simulants into the first MM state, letting them trickle through to the later RRMM states over 10 years, and then recording the prevalence of each MM/RRMM state as well as the risk exposure distribution at each state. The second burn in period will use the info from the first and will initialize simulants into MM and RRMM states with the corresponding risk exposure distributions but in 2016 so that all simulants will be initialized prior to dara/isa introduction into the treatment landscape. We'll then run the burn in period so that we can track how dara/isa exposure evolves over time as the simulants who have been treated survive/die and the background coverage increases over time.

Does this help clarify?